qiskit/transpiler/passes/synthesis/perm_row_col_synthesis.py

```diff
@@ -10,17 +10,15 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Perm_row_col function implementation"""
+"""Perm_row_col synthesis implementation"""
 
 import numpy as np
-import retworkx as rx
 
 from qiskit.transpiler.passes.synthesis.high_level_synthesis import HighLevelSynthesis
-from qiskit.circuit.library.generalized_gates.linear_function import LinearFunction
 from qiskit.dagcircuit.dagcircuit import DAGCircuit
+from qiskit.transpiler.synthesis.permrowcol import PermRowCol
 from qiskit.transpiler import CouplingMap
-from qiskit.converters import circuit_to_dag, dag_to_circuit
-from qiskit import QuantumRegister, QuantumCircuit
+from qiskit.converters import circuit_to_dag
 
 
 class PermRowColSynthesis(HighLevelSynthesis):
```

Comment:
I think Arianne wanted us to change the inheritance a bit so that *PermRowColSynthesis* is now a subclass of *LinearFunctionSynthesis*

Reply:
I know, but I considered that a separate task.